### PR TITLE
Lp 24 form web mobile new issue

### PR DESCRIPTION
**Pull request by fimak**
If user is logged in he will be redirected to `/` if he will try to open '/home' or /signup/ or '/signin'

### CODE REVIEW

File: src/Components/SignUp/CreateAnAccount/CreateAnAccount.jsx [hunk 1]
```diff
@@ -38,6 +39,8 @@ const CreateAnAccount = () => {
   const [windowstate, setWindow] = useState(window.innerWidth < 767);
   const { redirectUrl } = useSelector((state) => state.authReducer);
 
+  useRedirectIfUserLoggedIn();
```
Comment (biswanathsahoo) [inline src/Components/SignUp/CreateAnAccount/CreateAnAccount.jsx]: @fimak This component is being used to add new wallet for the logged in user. So for now let's not add the restriction on this page.

**Review by biswanathsahoo — CHANGES_REQUESTED** [review]

File: src/Components/SignUp/CreateAnAccount/CreateAnAccount.jsx (hunk 1 above)
Comment (idougwarner) [reply src/Components/SignUp/CreateAnAccount/CreateAnAccount.jsx]: @fimak I think we could replace the current route with the to route on the following pages. check this API reference - https://reactrouter.com/docs/en/v6/api#usenavigate

- when navigating from /signup page to /signup/create-account page
- when navigating from /signup/create-account page page to / page
- when navigating from /signin page to /signin/authentication/idougwarnerdev page
- when navigating from /signin/authentication/idougwarnerdev page to / page

then the user will be taken straight to the home page if the user clicks the go back browser button.

let's remove "Get Started" button and "Launch" buttons on the home page if the user session exists. Instead, let's put "Dashboard" button in the same location as the Launch button

Comment (idougwarner) [reply src/Components/SignUp/CreateAnAccount/CreateAnAccount.jsx]: @biswanathsahoo thought?

Comment (biswanathsahoo) [reply src/Components/SignUp/CreateAnAccount/CreateAnAccount.jsx]: Yes, this will be great

Comment (idougwarner) [reply src/Components/SignUp/CreateAnAccount/CreateAnAccount.jsx]: @fimak can you give me an update on this PR?

Comment (fimak) [reply src/Components/SignUp/CreateAnAccount/CreateAnAccount.jsx]: just a few minutes please

Comment (sparkyrider) [reply src/Components/SignUp/CreateAnAccount/CreateAnAccount.jsx]: @fimak is this one resolved?

**Review by biswanathsahoo — CHANGES_REQUESTED** [review]
@fimak Everything looks good except the below scenario 
If the user going to /signup/ page the it is redirecting to home page but if the user going to  /signup page then it is redirecting to /signup/create-account. This is because of the trailing slash at the end of the url slug. So consider   /signup/ is same as /signup

**Comment by fimak** [comment]
> @fimak Everything looks good except the below scenario If the user going to /signup/ page the it is redirecting to home page but if the user going to /signup page then it is redirecting to /signup/create-account. This is because of the trailing slash at the end of the url slug. So consider /signup/ is same as /signup

Fixed

File: src/Components/SignUp/CreateAnAccount/CreateAnAccount.jsx (hunk 1 above)
Comment (fimak) [reply src/Components/SignUp/CreateAnAccount/CreateAnAccount.jsx]: @sparkyrider 
Yes

**Review by biswanathsahoo — APPROVED** [review]

**Pull request merged**